src/main/java/cz/muni/fi/pv286/mmt/exceptions/InvalidBitCharacterException.java

```diff
@@ -0,0 +1,16 @@
+package cz.muni.fi.pv286.mmt.exceptions;
+
+import java.io.IOException;
+
+/**
+ * This exception is used for invalid bit characters.
+ */
+public class InvalidBitCharacterException extends IOException {
+    public InvalidBitCharacterException(String message) {
+        super(message);
+    }
```

Comment:
**suggestion**: Remove unused function.

It's a code smell and the code it's harder to read.